src/main/java/org/spongepowered/api/entity/living/Human.java

```diff
@@ -119,6 +151,37 @@
     void setTotalExperience(double totalExperience);
 
     /**
+     * Returns whether this human entity is sneaking.
+     *
```

Comment:
Possibly mention in a new paragraph that name tags may have decreased visibility, movement may be slowed etc.